tests/network/primary_pod_network.go

```diff
@@ -88,13 +88,13 @@ var _ = SIGDescribe("[Serial]Primary Pod Network", func() {
 				var vmi *v1.VirtualMachineInstance
 
 				BeforeEach(func() {
-					tmpVmi, err := newFedoraWithGuestAgent()
+					var err error
+					vmi, err = newFedoraWithGuestAgent()
 					Expect(err).NotTo(HaveOccurred())
 
-					tmpVmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(tmpVmi)
+					vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
 					Expect(err).NotTo(HaveOccurred())
-					vmi = tests.WaitUntilVMIReady(tmpVmi, tests.LoggedInFedoraExpecter)
-
+					tests.WaitForSuccessfulVMIStart(vmi)
```

Comment:
this test changes can be reverted right ? 
you adapted other test

Reply:
Done